tests/functional/pages/surveys/census/household/address-type.page.js

```diff
@@ -2,6 +2,11 @@ import MultipleChoiceWithOtherPage from '../../multiple-choice.page'
 
 class AddressTypePage extends MultipleChoiceWithOtherPage {
 
+  isOpen() {
```

Comment:
Can this not move up the the question page class and then just provide the blockId in each file?